ledger/light/artifactmanager/handler.go

```diff
@@ -270,7 +270,7 @@ func (h *MessageHandler) setHandlersForLight(m *middleware) {
 	h.Bus.MustRegister(
 		insolar.TypeGetPendingRequestID,
 		BuildMiddleware(
-			h.handleGetPendingRequestID,
+			h.FlowDispatcher.WrapBusHandle,
```

Comment:
This piece of code looks funny. I don't believe we keep middlewares during migrations to the flow. The handler has to contain the code of checkJet.

Reply:
In fact IIRC getFetchJet already does this thus you can get rid of middlewares here.